pom.xml

```diff
@@ -203,7 +205,16 @@
                     <configuration>
                         <source>${project.build.targetJdk}</source>
                         <target>${project.build.targetJdk}</target>
+                        <!-- configure for flink lambda support -->
```

Comment:
this change should not be made during a PR for Accumulo support. this should be a PR on its own and tested as well. Not sure if we even want to use this right no. I have no experience with it. What's you suggestion @s1ck ?

Reply:
I agree. @chpengzh is this necessary for this PR?

Reply:
Seems to be unnessary here. 

I found this some days before. I wrote another project which depend on gradoop. I import the whole project (in order to learn api), and decide to implement a transE alg demo. I found there's something wrong with flink lambda support.

```java
<project.build.targetJdk>1.8</project.build.targetJdk>
```

I'll remove this, or should I will create another issue to describe this after works done

Reply:
removed in [7819962](https://github.com/dbs-leipzig/gradoop/pull/759/commits/781996277bcbd853b537f8ddde9de2aa4ca8fffa)